pxr/usdImaging/usdviewq/appController.py

```diff
@@ -2172,29 +2175,74 @@ def setFrameField(self, frame):
 
     # Prim/Attribute search functionality =====================================
 
+    def _isMatch(self, pattern, isRegex, prim, useDisplayName):
```

Comment:
A _suggestion_ that if `_isMatch` were defined inside of `_findPrims`, then it would only need a `prim` parameter.  Perhaps you were anticipating other uses outside of _findPrims, and there's also something to be said for having documented parameters, but thought we'd mention it for your consideration.

Reply:
I would leave this to your preference, as I cannot attest to whether it might be used elsewhere.  I can move it if you prefer.

Reply:
Let's leave it as it is, for now, then.